opencolorio_config_aces/clf/transforms/arri/input/ARRI.Input.ARRI_LogC3_Curve_EI800.clf

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ProcessList compCLFversion="3" id="urn:aswf:ocio:transformId:1.0:ARRI:Input:ARRI_LogC3_Curve_EI800_to_Linear:1.0" name="ARRI LogC3 Curve (EI800) to Relative Scene Linear">
+    <InputDescriptor>ARRI LogC3 Curve (EI800)</InputDescriptor>
+    <OutputDescriptor>Relative Scene Linear</OutputDescriptor>
```

Comment:
No strong objection, but we should try to ensure that the descriptions are clear in terms of what is output.  We certainly want to avoid the situation where people mistakenly think the output is ACES2065-1.

As you know, this curve will become a NamedTransform and it will just apply the linearization, leaving the output in the same primaries/gamut as the input.  Typically this will be AWG, but in practice it could be anything since people could send anything into the transform.

In the other similar curve CLFs I used the same words for both input and output except for using "Log" in the input and "Linear" in the output.  No need to copy that, I just wanted to mention it as a point of comparison.

I agree that "Relative Scene Linear" is more precise than just "Linear".  But I used just "Linear" since it made it easier to use mostly the same words for both input and output.

Reply:
I'm a bit confused by this comment. The CLF doesn't reference primaries/gamut at all, so I don't see how ACES2065-1 would be implied.

Reply:
Right, "Relative Scene Linear" does not state the primaries.  That's why I felt less experienced people may make an assumption that was incorrect.  But I think we had sufficient discussion about this during the meeting.  I have no objection to leaving it as is.